src/itemset.js

```diff
@@ -201,30 +227,21 @@ define([
     }
 
     function itemsetWidget(mug, options) {
-        var widget = datasources.dataSourceWidget(mug, options, "Data Source"),
+        var widget = datasources.fixtureDataSourceWidget(mug, options, "Lookup Table"),
             super_getUIElement = widget.getUIElement,
             super_getValue = widget.getValue,
-            super_setValue = widget.setValue,
-            handleChange = widget.handleChange.bind(widget),
-            labelRef = refSelect("label_ref", "Choice Label", widget.isDisabled()),
-            valueRef = refSelect("value_ref", "Choice Value", widget.isDisabled());
-
-        labelRef.onChange(handleChange);
-        valueRef.onChange(handleChange);
+            super_setValue = widget.setValue;
 
         widget.getUIElement = function () {
-            return super_getUIElement()
-                .append(labelRef.element)
-                .append(valueRef.element);
+            return super_getUIElement();
         };
```

Comment:
seems like this function is no longer necessary since it's a straight pass-through now